LEGO1/lego/legoomni/src/actors/skateboard.cpp

```diff
@@ -17,17 +20,44 @@ SkateBoard::SkateBoard()
 	NotificationManager()->Register(this);
 }
 
-// STUB: LEGO1 0x10010000
+// FUNCTION: LEGO1 0x1000ff80
+SkateBoard::~SkateBoard()
+{
+	ControlManager()->Unregister(this);
+	NotificationManager()->Unregister(this);
+}
+
+// FUNCTION: LEGO1 0x10010000
 MxResult SkateBoard::Create(MxDSAction& p_dsAction)
 {
-	// TODO
-	return SUCCESS;
+	MxResult result = IslePathActor::Create(p_dsAction);
+	if (result == SUCCESS) {
+		this->m_world = CurrentWorld();
+		this->m_world->Add(this);
+		// The type `Pizza` is an educated guesss, inferred from VTable0xe4() below
+		Pizza* findResult = (Pizza*) CurrentWorld()->Find(*g_isleScript, 0x49d);
+		if (findResult) {
+			findResult->m_unk0x84 = (undefined*) this;
+		}
+	}
+	return result;
 }
 
-// STUB: LEGO1 0x10010050
+// FUNCTION: LEGO1 0x10010050
 void SkateBoard::VTable0xe4()
 {
-	// TODO
+	// TODO: Work out what kind of structure this points to
```

Comment:
The hardcoded hex values should also be replaced with the enum constants from the script header; see the other comment